### DISCUSSION TITLE
Welcome to mlx-examples Discussions!

### DISCUSSION THREAD

**Discussion by awni** [ml-explore/mlx-examples#769] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fblissjr** [comment]
Hi all, I've been actively following mlx since I first saw it released - in fact, it's the top reason I ended up buying a mac, after 40+ years of never having touched one. I think it's the most promising path forward for on-device AI usage - especially given the large number of tasks that can be handled locally on low compute/ram, that are currently offloaded to cloud / API providers.

This has been a great community to learn from and actively watch get better each day. The passion and curiosity reminds me a lot of the 90s dot com cycle. 

Thanks to everyone who keeps contributing to this project and making it better. I certainly make use of it daily.

**Reply by DeltaFROST141** [reply]
This is the meaning of open sources ~
